androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/TwaLauncher.java

```diff
@@ -107,28 +115,29 @@ void launch(Context context,
      * Creates an instance that will automatically choose the browser to launch a TWA in.
      * If no browser supports TWA, will launch a usual Custom Tab (see {@link TwaProviderPicker}.
      */
-    public TwaLauncher(Context context) {
-        this(context, null);
+    public TwaLauncher(Context context, long startupUptimeMillis) {
```

Comment:
I think these are public APIs, and adding a new required parameter breaks API compatibility.

I think we might want to grab the timestamp in launch() instead for now, should be close enough.

Reply:
Good catch about the public API! Normally what we'd do in this case is add a new overload that takes the value and then make the old version call that overload with a default value.

However, at the moment, both the constructor and `launch` have quite a few overloads already so things would get quite messy.

It's probably simplest to just have a `setStartupTimeMillis` method on `TwaLauncher`.

Reply:
Thanks for the review, I added `setStartupTimeMillis` in `TwaLauncher`.